.github/workflows/ci.yml

```diff
@@ -4,10 +4,64 @@ on:
   pull_request:
     branches: [main]
   schedule:
-    - cron: "0 0 * * *" # Run every day at 00:00 UTC.
+    - cron: "2 0 * * *" # Run every day at 02:00 UTC.
   workflow_dispatch:
 
+env:
+  REGISTRY_USER: ${{ github.actor }}
+  REGISTRY_PASSWORD: ${{ github.token }}
+  IMAGE_REGISTRY: ghcr.io/${{ github.repository_owner }}
+  # This is required for running GUI tests on Github Actions
+  DISPLAY: ":99.0"
+  QT_SELECT: "qt6"
+
+# Disable multiple concurrent runs on the same branch
+# When a new CI build is triggered, it will cancel the
+# other in-progress ones (for the same branch)
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
 jobs:
+  run-lint:
+    runs-on: ubuntu-latest
+    container:
+      image: debian:bookworm
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dev. dependencies
+        run: |-
+          apt-get update
+          apt-get install -y git make python3 python3-poetry --no-install-recommends
+          poetry install --no-ansi --only lint,test
```

Comment:
Is the `--no-ansi` flag here needed anymore? The issue mentioned in the circleci config got fixed in cleo 2.1 which was included in poetry 1.7 released last November.

Reply:
The quick answer here is "probably not". But I'm more interested in how you stumbled on this. Did it mess with the logs in your CI runs?

Reply:
It didn't. Just happened to notice it while poking around the linters used in the Dangerzone repo.

Once I'm done with the windows installer, I'm planning on replacing black and isort with ruff for code formatting and linting as my next small project in the dangerzone codebase. Mostly since theres the issue about unused imports still open and I noticed Securedrop made the change a while ago and thought why not do that here too? A good excuse to play around with ruff, something I've wanted to do for a while now, and Dangerzone repo benefits from faster linting with potential to enable a ton of linting tools in one with little effort.

Though, first I'll open an issue proposing the change in a couple of days.

Reply:
Thanks for the remark, this is fixed in 510c169b0baa3ae4a3dc99d480c7836779c81c81